flake8_quotes/__init__.py

```diff
@@ -29,13 +30,35 @@ class QuoteChecker(object):
     def __init__(self, tree, filename='(none)', builtins=None):
         self.filename = filename
 
+    @staticmethod
+    def _register_opt(parser, *args, **kwargs):
+        """
+        Handler to register an option for both flake8 version 3 and version 2.
+
+        This is based on:
+        http://flake8.pycqa.org/en/latest/plugin-development/cross-compatibility.html
```

Comment:
This link current points to `/latest` which could easily break. Maybe we should point to a tagged GitHub document:

https://github.com/PyCQA/flake8/blob/3.0.0b2/docs/source/plugin-development/cross-compatibility.rst#option-handling-on-flake8-2-and-3